### DISCUSSION TITLE
INI was not able to parse expert_execution

### DISCUSSION THREAD

**Discussion by Kazusa-Touma** [srsran/srsRAN_Project#345] [General Help]
I am trying to modify gNB configuration file, and when I try to add configuration parameters like affinity and threads according to [this](https://docs.srsran.com/projects/project/en/latest/user_manuals/source/config_ref.html), it shows INI was not able to parse expert_execution
![屏幕截图 2023-11-30 140535](https://github.com/srsran/srsRAN_Project/assets/62680205/73f71988-e35f-40b8-96e1-5d686bd02b1b)

**Comment by andrepuschmann** [comment]
This section is still subject to change and there is likely a mismatch between the versions you're using. You're not saying which version you use but just try to upgrade to the latest or the last official release and give this a try.
